### PR TITLE
Add shop to logged in dropdown

### PR DESCRIPTION
**Pull request by Robinnnnn**
Update (OBJECTS) to use proper brackets

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -108,5 +108,6 @@
     "ts-jest": "^27.0.7",
     "ts-prune": "^0.10.3",
     "typescript": "^4.7.4"
-  }
+  },
+  "version": "0.0.0"
```
Comment (Robinnnnn) [inline package.json:112]: github being confused

File: src/contexts/globalLayout/GlobalFooter/GlobalFooter.tsx [hunk 2]
```diff
@@ -43,6 +45,16 @@ function GlobalFooter() {
...
+          <StyledFooterLink href="/shop">
+            Shop
+            <Spacer width={4} />
+            <StyledObjectsContainer>
+              <StyledLogoBracketLeft color={colors.shadow} />
+              <StyledShopText>OBJECTS</StyledShopText>
+              <Spacer width={1} />
+              <StyledLogoBracketRight color={colors.shadow} />
+            </StyledObjectsContainer>
+          </StyledFooterLink>
```
Comment (Robinnnnn) [inline src/contexts/globalLayout/GlobalFooter/GlobalFooter.tsx:57]: image

File: src/contexts/globalLayout/GlobalNavbar/LoggedInNav.tsx [hunk 3]
```diff
@@ -96,6 +99,17 @@ function LoggedInNav({ queryRef }: Props) {
...
+              <ShopOptionContainer>
+                <StyledShopTextButton text="shop" />
+                <StyledObjectsContainer>
+                  <StyledLogoBracketLeft color={colors.shadow} />
+                  <StyledShopText>OBJECTS</StyledShopText>
+                  <Spacer width={1} />
+                  <StyledLogoBracketRight color={colors.shadow} />
+                </StyledObjectsContainer>
+              </ShopOptionContainer>
+            </UnstyledLink>
```
Comment (Robinnnnn) [inline src/contexts/globalLayout/GlobalNavbar/LoggedInNav.tsx:112]: image

File: src/scenes/LandingPage/LandingPage.tsx [hunk 4]
```diff
@@ -41,9 +44,17 @@ export default function LandingPage() {
...
+            Shop
+            <Spacer width={6} />
+            <StyledObjectsContainer>
+              <StyledLogoBracketLeft color={colors.shadow} />
+              <StyledShopText>OBJECTS</StyledShopText>
+              <StyledLogoBracketRight color={colors.shadow} />
+            </StyledObjectsContainer>
+            <Spacer width={6} />
+            <BlueLabel>New</BlueLabel>
+          </StyledShopLinkContainer>
```
Comment (Robinnnnn) [inline src/scenes/LandingPage/LandingPage.tsx:57]: image

File: src/scenes/MerchStorePage/MerchStorePage.tsx [hunk 5]
```diff
@@ -40,8 +44,12 @@ export default function MerchStorePage() {
...
     <StyledPage>
       <Countdown />
-      {/* TODO: Use brackets from Gallery Logo here */}
-      <StyledShopText>( OBJECTS )</StyledShopText>
+      <StyledLogoContainer>
+        <StyledLogoBracketLeft color={colors.offBlack} />
+        <StyledShopText>OBJECTS</StyledShopText>
+        <Spacer width={1} />
+        <StyledLogoBracketRight color={colors.offBlack} />
+      </StyledLogoContainer>
```
Comment (Robinnnnn) [inline src/scenes/MerchStorePage/MerchStorePage.tsx:52]: image

File: package.json (hunk 1 above)
Comment (tbezman) [inline package.json:112]: was this intentional?

**Pull request merged**